app/views/admin/index.html.erb

```diff
@@ -1,17 +1,8 @@
-<%= react_component("admin/Admin", {
-    data: User.all.where(jurisdiction_id: current_user.jurisdiction.subtree_ids)
-        .joins(:jurisdiction)
-        .select('users.id, users.email, users.api_enabled, users.locked_at, users.authy_id, jurisdictions.path')
-        .map {
-            |u| u.attributes.slice('id', 'email', 'api_enabled').merge(
-                jurisdiction_path: u.path, 
-                role: u.roles[0].name,
-                locked: !u.locked_at.nil? ? 'Locked' : 'Unlocked', 
-                configured_2fa: !u.authy_id.nil? ? 'Yes' : 'No'
-            )
-    },
-    role_types: Role.pluck(:name).uniq.reverse, 
-    authenticity_token: form_authenticity_token, 
-    jurisdiction_paths: Hash[Jurisdiction.where(id: current_user.jurisdiction.subtree_ids).map{|jur| [jur[:path], jur.id]}], 
-    is_usa_admin: current_user.can_send_admin_emails? })
- %>
+<%= stylesheet_link_tag "AdminTable" %>
+<%= react_component("admin/AdminTable", {
+        role_types: Role.pluck(:name).uniq.reverse.map{|role| role.split('_').map(&:capitalize).join(' ')}, 
+        authenticity_token: form_authenticity_token, 
+        jurisdiction_paths: Hash[Jurisdiction.where(id: current_user.jurisdiction.subtree_ids).map{|jur| [jur[:path], jur.id]}], 
```

Comment:
So that endpoint action is blocked for admins because of the `authenticate_user_role` check that happens before all actions in the `jurisdictions_controller`. It requires that the user can either create a patient, edit a patient, or view the public health dashboard which doesn't apply to admins. I added a similar endpoint to the `admin_controller` for now as I don't think that we that we want to change the permissions on the jurisdiction routes. Let me know what you think.

Reply:
I think we can definitely expand the permissions on that jurisdictions route to include admins because admins should and already have access to that information

Reply:
Good point, done.